### DISCUSSION TITLE
UCAN Community Website Kick off Call Dec 13th, 2022

### DISCUSSION THREAD

**Discussion by bmann** [ucan-wg/spec#138] [Community Call]
Thread for follow ups

Repo: https://github.com/ucan-wg/ucan-community-web

Discord channel: https://discord.com/channels/1002414684579827792/1044692478609072179

**Comment by bmann** [comment]
@bmann 

@bgins 

Design
Content

Joel:
* could be a learning experience -- too much text
* love the tools
* UCAN validator and other tools front and center

JChris
* we already have the depth! yay more tools
* get people to come back
* it should be a verb rather than a noun
* more tools -- place to get all of them
* delegation composer
* verb is showing off that there are power features, regardless of whether the developer made that
* if we want to raise awareness
* Raft / Consensus -- learning UCAN
* shows up on threads for computer programmers several times per year
* UCAN are discovered not invented
* "crypto signatures are inevitable"
* https://docs.google.com/presentation/d/1BKY1SV88vbT7W8rZojNrfapdWMS0Tsd9WL0Tn9uLv7g/edit?usp=sharing

CData
* https://www.passkeys.io/ as inspiration

Awareness
Learning 
Adoption
Ecosystem
- wall of logos

UCAN Registry
- type management
- care about own types
- Postel's law
- TypeScript headers
- useful for validator

Cdata: generalized capability definition
JChris: spaces at this level, "free plan"

Boris: "UCAN as IAM for the Open Web"

Cdata: Web3Storage has most useful delegated service today
Knowing that a thing exists
Can I store something somewhere?

UCAN Adventure - https://ucan.lol
* curl or Postman
* certificate of completion

JChris: DAG House designer

Becky: UCAN Video script
* explainer video
* what else should it do?
* CData: for some people -- what makes UCAN special?
* analog for OAuth - call out, like this other thing - single sign in with Apple, Google

Brian:
* Intro to GunDB https://www.youtube.com/watch?v=oTQXzhm8w_8
Link to all of the implementations

CData: Kubo syndication -- need to figure it out wi

Becky: how to edit via Github?
* bgins to do a walkthrough for people who want to learn

JChris: OpenLinks
* voice over on Name Name
* MongoDB - MEAN stack Mongo Express Angular Node
* what's the stack? UCAN, IPFS, IPLD, 

DAG Summit
* inviting folks hang out in Portland
* Unconf
* yes, Causal Islands is confirmed Apr 25 - 27th in Toronto
* IPFS thing either week before CI, or march 27th week around Paris
* May 1st - May 5th - save the date!

**Reply by meghansinnott** [reply]
Here's a list of web3.storage-related UCAN resources:
https://blog.web3.storage/posts/ucan-delegation-with-w3up
https://www.youtube.com/watch?v=7OznUYYhC1o&list=PLuhRWgmPaHtSEv2DM9J7nSvN7dnVFMM6q&index=89&t=17s
https://blog.web3.storage/posts/recap-web3-storage-in-lisbon
https://blog.web3.storage/posts/intro-to-ucan
https://www.notion.so/UCAN-talk-a2267c87fb3248d1a6a55beea971bbd8
DRAFT - https://hackmd.io/XgHeooJ1RrC5wuJ_2upGiw?view

**Reply by bmann** [reply]
Thanks @meghansinnott !
